test/fw/ptl/utils/pbs_dshutils.py

```diff
@@ -184,6 +184,10 @@ def get_platform(self, hostname=None, pyexec=None):
             else:
                 splatform = 'craysim'
             found_already = True
+        if self.isfile(hostname=hostname, path='/etc/cray/xname',
+                       level=logging.DEBUG2):
```

Comment:
This if also needs a `if not found_already`

Reply:
found_already should not be true since we check for cray_xt before this check. So I did not see a need for that extra check.